### PR TITLE
Removed previous signup api

### PR DESCRIPTION
**Pull request by khaniku**

### CODE REVIEW

File: src/actions/api.js [hunk 1]
```diff
@@ -23,32 +23,6 @@ export function login(user, client){
     });
 }
 
-// export function signup(user){
```
Comment (khaniku) [inline src/actions/api.js:26]: Why didn't you use axis?

File: src/actions/api.js [hunk 2]
```diff
@@ -23,32 +23,6 @@ export function login(user, client){
...
-//     .then((response) => response.json())
-//     .then((responseJson) => {
-//         console.log(responseJson)
-//         return responseJson;
-//     })
-//     .catch((error) => {
-//         console.log(error);
-//     });
-// }
-
```
Comment (khaniku) [inline src/actions/api.js:51]: Did you consider using GraphQL?

**Review by khaniku — COMMENTED** [review]
This is just a test

File: src/actions/api.js (hunk 1 above)
Comment (khaniku) [reply src/actions/api.js:26]: modified!

**Review by unicorncodak — APPROVED** [review]

**Pull request merged**